src/briefcase/commands/new.py

```diff
@@ -307,12 +307,7 @@ def validate_url(self, candidate):
         :returns: True. If there are any validation problems, raises ValueError with a
             diagnostic message.
         """
-        result = urlparse(candidate)
-        if not all([result.scheme, result.netloc]):
-            raise ValueError("Not a valid URL!")
-        if result.scheme not in {"http", "https"}:
-            raise ValueError("Not a valid website URL!")
-        return True
+        return validate_url(candidate)
```

Comment:
This is method named `validate_url()` that is invoking a function named `validate_url()` with the same arguments; the extra layer of the method isn't adding anything, so it can be removed. Any usage of `self.validate_url()` can be replaced with a direct `validate_url()` invocation.